tails_files/install.sh

```diff
@@ -61,14 +83,19 @@ if $ADMIN; then
   echo "# HidServAuth lines for SecureDrop's authenticated hidden services" | cat - $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths > $ADDITIONS
   if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
     # create SSH host aliases and install them
-    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
+    # parse prod-specific.yml YAML into Bash variables
+    eval $(parse_yaml $ANSIBLE/prod-specific.yml)
+    if [[ -z $ssh_users ]]; then
+        echo "Error: ssh_users is not defined in prod-specific.yml." 1>&2
+        exit 1
+    fi
     cat > $INSTALL_DIR/ssh_config <<EOL
 Host app
   Hostname $APPSSH
-  User $SSHUSER
+  User $ssh_users
 Host mon
   Hostname $MONSSH
-  User $SSHUSER
+  User $ssh_users
```

Comment:
@garrettr Ansible will Do The Right Thing with `ssh_users`, since the associated tasks are already using `with_items` (see `roles/common/tasks/create_users.yml`). If the value of `ssh_users` is a string, rather than a list, then Ansible will interpret it as a list with length of 1 while processing `with_items`. If it's a YAML list, then it'll be a list of length *n*, where *n* is the number of usernames provided.

You can check this by updating the vars file and running `ANSIBLE_ARGS="-t users" vagrant provision /staging/`. The sudoers config is a static file, and not a template, so doesn't change regardless of how many admin users are created. The tasks that create the admin users add the usernames to the `ssh` and `sudo` groups, which makes sudo work correctly for all accounts.

Therefore I think it's appropriate to leave the variable named as `ssh_users`, since it does indeed accommodate multiple usernames.